### PR TITLE
[4.0] Update the com_content manifest file

### PR DESCRIPTION
**Pull request by laoneo**
Pull Request for pr #21790.

### Summary of Changes
As #21790 revealed, the manifest files in core are out of sync. Before I'm going to fix all of them I would like to get some feedback what all needs to be adapted.

Beside the folders I would also change the compatibility version and the version of the extension itself. Anything else?

### CODE REVIEW

File: administrator/components/com_content/content.xml [hunk 1]
```diff
@@ -24,14 +30,18 @@
 		<files folder="admin">
 			<filename>access.xml</filename>
 			<filename>config.xml</filename>
-			<filename>content.php</filename>
-			<filename>controller.php</filename>
-			<folder>controllers</folder>
-			<folder>elements</folder>
+			<folder>Controller</folder>
```
Comment (brianteeman) [inline administrator/components/com_content/content.xml]: missing 
filename content.xml

**Comment by brianteeman** [comment]
other than the missing file it looks ok to me

would be good to get another check though

File: administrator/components/com_content/content.xml [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
 		<filename>router.php</filename>
+		<folder>Controller</folder>
+		<folder>Dispatcher</folder>
+		<folder>forms</folder>
+		<folder>Helper</folder>
 		<folder>helpers</folder>
-		<folder>models</folder>
+		<folder>Model</folder>
+		<folder>Service</folder>
+		<folder>Table</folder>
```
Comment (alikon) [inline administrator/components/com_content/content.xml]: if i'm not wrong there is no Table folder

**Comment by laoneo** [comment]
Added also media folder.

**Comment by brianteeman** [comment]
I am wondering if we even need these manifest files for core components and with the move of media files if they should be in the manifest either

**Comment by mbabker** [comment]
Yes the extension manifests should be in place.  Technically, we don't need them, but the core package shouldn't be taking liberties as it relates to extension structure that a distributed extension can't.

**Comment by brianteeman** [comment]
And the media folder? The same?

@laoneo did you have to create this manually - if so I am happy to takeover so you can concentrate on developer stuff

**Review by wilsonge — CHANGES_REQUESTED** [review]
I don't think we need the media tag - we aren't shipping a media folder with the extension as it's all baked into core (especially now with the es6 stuff) - so I'd leave that as it is. Otherwise LGTM

**Comment by laoneo** [comment]
For me it reflects the installed state of an extension which includes the media folder and com_content has a media folder. From my understanding an extension dev should be able to take it as example and the manifest with the media folder is part of it.

I can remove it if you really want but I think it is not a bad idea to leave it in.

@brianteeman yes I'm doing them by hand. If you want you can take over the task when we get this one merged. But I would wait about the agreement of the media folder.

**Comment by brianteeman** [comment]
When you decide on media let me know and i can get this done

**Comment by wilsonge** [comment]
Not convinced about the media tag - but not sceptical enough to block progress here

**Pull request merged**